src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/Unix/PkcsPal.Unix.cs

```diff
@@ -3,17 +3,209 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Buffers;
 using System.Diagnostics;
+using System.Security.Cryptography;
+using System.Security.Cryptography.Asn1;
+using System.Security.Cryptography.Pkcs;
+using System.Security.Cryptography.X509Certificates;
 
 namespace Internal.Cryptography
 {
     internal abstract partial class PkcsPal
     {
-        private static PkcsPal s_instance;
+        private static PkcsPal s_instance = new ManagedPkcsPal();
 
-        static PkcsPal()
+        private class ManagedPkcsPal : PkcsPal
```

Comment:
This class is platform-agnostic, no?I think you should either:

1. Rename class to PkcsPal.Managed.cs. Keep your current netcoreapp configuration in the csproj and remove the unix references.
2. Keep class as PkcsPal.Unix.cs. Rename your netcoreapp configuration in the csproj to netcoreapp-unix

Reasoning is that it's incorrect to have a platform-agnostic configuration that includes a "*.Unix.cs" file, though in this case you're fine since the file isn't actually Unix specific (from what I see).

Reply:
Renamed to `AnyOS\PkcsPal.AnyOS.cs`